target/linux/bmips/patches-5.15/203-mips-bmips-dma-fix-CBR-address.patch

```diff
@@ -66,11 +66,12 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
 -	 * RAC flush causes kernel panics on BCM6358 when booting from TP1
 -	 * because the bootloader is not initializing it properly.
 -	 */
--	bmips_rac_flush_disable = !!(read_c0_brcm_cmt_local() & (1 << 31));
+-	bmips_rac_flush_disable = !!(read_c0_brcm_cmt_local() & (1 << 31)) ||
+-				  !!BMIPS_GET_CBR();
```

Comment:
@Ansuel  Should I remove this patch?
This patch was backported: https://github.com/gregkh/linux/commit/89167072fd249e5f23ae2f8093f87da5925cef27